starter_kits/C/src/MyBot.c

```diff
@@ -0,0 +1,324 @@
+// First off, here are some defines and includes. I forget what exactly needs what
+#include <stdio.h>
+#include <stdlib.h>
+
+// We include time.h so we can seed the RNG for default "smart" movement with time()
+#include <time.h>
+
+
+
+// Now for our data structures. These are intentionally left rather basic, so add whatever you want to them
+struct ship{
+	unsigned char x, y;
+	unsigned short halite;
+} ships[4096];
+
+struct player{
+	unsigned char dropoffs[64], dropoff_count, ship_count, ship_yard[2];
+	unsigned short ships[1024];
+	unsigned int halite;
+} players[4];
+
+struct map{
+	unsigned short halite;
+	unsigned short occupied; //(unsigned short)-1 means it's not occupied
+} map[64][64];
```

Comment:
I'm confused on how these buffer sizes are chosen.
Why `[64][64]`?

Reply:
https://halite.io/learn-programming-challenge/game-overview
Ah, Max map size is 64.